packages/@guidesmiths/cuckoojs-schematics/src/helm/helm-strategies/tls-cert/cert-manager.tlsCert.strategy.ts

```diff
@@ -0,0 +1,8 @@
+import {TlsCertStrategy} from './tlsCert.strategy';
+
+export class CertManagerTlsCertStrategy extends TlsCertStrategy {
+	constructor() {
+		super(true);
+		this.templatePath.push('cert-manager');
```

Comment:
Is it possible to declare an enum in ingress.strategy.factory.ts with the `none` and `certManager` values to avoid hardcoded values?